core/src/rpc.rs

```diff
@@ -514,6 +514,22 @@ impl JsonRpcRequestProcessor {
             Ok(None)
         }
     }
+
+    pub fn get_address_confirmed_signatures(
+        &self,
+        pubkey: Pubkey,
+        start_slot: Option<Slot>,
+        end_slot: Option<Slot>,
```

Comment:
I think we should make start/end slot required, and cap the difference between them to some limit.  Maybe 10,000 slots (~1 hour).

App devs are lazy and they'll just not provide any start/end slot and the node'll get crushed with all the `(0, std::u64::MAX)` queries

Reply:
Fair enough